hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/AdamicAdarAPI.java

```diff
@@ -80,9 +80,10 @@ public String get(@Context GraphManager manager,
         Directions dir = Directions.convert(EdgeAPI.parseDirection(direction));
 
         HugeGraph g = graph(manager, graph);
-        PredictionTraverser traverser = new PredictionTraverser(g);
-        double score = traverser.adamicAdar(sourceId, targetId, dir,
-                                            edgeLabel, maxDegree, limit);
-        return JsonUtil.toJson(ImmutableMap.of("adamic_adar", score));
+        try (PredictionTraverser traverser = new PredictionTraverser(g)) {
+            double score = traverser.adamicAdar(sourceId, targetId, dir,
+                edgeLabel, maxDegree, limit);
```

Comment:
prefer to keep the origin alignment

Reply:
if one line < 100 chars, use one line is fine (and we have a code-style setting file in root for IDEA, use it to modify code is much more easy)

Reply:
fixed the alignment. Thanks for the tip on code-style setting file.